pinot-core/src/test/java/org/apache/pinot/core/segment/store/ColumnIndexDirectoryTestHelper.java

```diff
@@ -60,11 +60,11 @@ static PinotDataBuffer getIndexBuffer(ColumnIndexDirectory columnDirectory, Stri
   static void verifyMultipleReads(ColumnIndexDirectory columnDirectory, String column, int numIter)
       throws Exception {
     for (int ii = 0; ii < numIter; ii++) {
```

Comment:
Might be good to rename the index variable?

Reply:
Done lol